objectRecognitionSource/src/main/java/edu/ml/tensorflow/ObjectDetector.java

```diff
@@ -127,6 +127,7 @@ public ObjectDetector(float thresh, String graphFile, String labelFile, String m
                 }
             }
             this.imageUtil = imageUtil;
+            imageUtil.frameSize = this.frameSize;
```

Comment:
I think this is Ok to get this done, but I'm confused (in reading it all) as to who's in charge of what.  We have, in ODCore, the setupImageUtil() that constructs the imageUtil, but, apparently, gets the frame size wrong.  So we go in after the fact and stuff in a new value.  Seems like that should be either a parameter that it gets at startup OR it should know to go elsewhere (maybe it's metaconfig) to find it.  

The problem with "lots of folks" poking various public things is that it's very hard to track down what's going, where, etc.  I realize we're attempting to deal with some inherited code here, having it deal with things it wasn't built to do, but it'll prove problematic in the long run.